contextCmds/test/msgCmd.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    name: "content",
+    type: 3, // 2 is for "USER" | 3 is for "MESSAGE" (EXECUTED BY RIGHT CLICKING EITHER)
```

Comment:
You have to explain this to me

Reply:
It's a context menu command (right clicking any message -> apps -> command)